uiowa/uiowa
# Panopto Support v2/HR

Closes #3691 & Closes #3676 


# How to test
* `blt frontend`
* As a webmaster on the brand.uiowa.edu, I want the ability to add/edit/delete organization terms as requirements change.
  * blt ds --site=brand.uiowa.edu
  * Make sure you can add/edit/delete organization taxonomy terms  as a webmaster
* As a v2/HR site editor/publisher/webmaster, I can embed Panopto videos.
   * blt ds --site=hawkeyemarchingband.uiowa.edu
   * blt ds --site=hr.uiowa.edu
   * Make sure you can add a UICapture (Panopto video) in WYSIWYG media as a editor/publisher/webmaster
* As a site editor/publisher/webmaster, the remote video/panopto video that I place in the WYSIWYG looks similar to what is rendered.
   *  blt ds --site=sandbox.uiowa.edu https://sandbox.prod.drupal.uiowa.edu/media
   *  https://github.com/uiowa/uiowa/issues/3725



## ToDo

- [x] Original (media--view-mode-default) in uiowa_bootstrap isn't rendering
- [x] Alignment without captions is not working for v2/v3
- [x] General HR support (preview/render)
- [x] Previews for Kaltura are broken
